lib/console_web/controllers/router/gateway_controller.ex

```diff
@@ -23,11 +24,16 @@ defmodule ConsoleWeb.Router.GatewayController do
     with gateway = %Gateway{} <- HardwareIdentifiers.get_resource_by_hardware_identifier(token, Gateway), # need to verify OUI
       "pending" <- gateway.status,
       {:ok, _} <- Gateways.update_gateway(gateway, %{status: "verified"}) do
+        broadcast(gateway, "verified")
 
         conn
         |> send_resp(:no_content, "")
     else _ ->
       {:error, :not_found}
     end
   end
+
+  defp broadcast(%Gateway{} = gateway, action) do
+    ConsoleWeb.Endpoint.broadcast("gateway:#{gateway.id}", action, %{})
```

Comment:
do we want to use the plain phoenix channels over graphql subscriptions?

Reply:
hmmm let me think about this one, didn't think to use it since we aren't fetching anything with graphql for GatewayRegister